pywbem/_cim_operations.py

```diff
@@ -394,7 +399,9 @@ class WBEMConnection(object):  # pylint: disable=too-many-instance-attributes
 
     After creating a :class:`~pywbem.WBEMConnection` object, various methods
     may be called on the object, which cause WBEM operations to be issued to
-    the WBEM server. See :ref:`WBEM operations` for a list of these methods.
+    the WBEM server or WBEM listener. See :ref:`WBEM operations` for a list of
```

Comment:
I don't see the export... method listed in that table.

Reply:
Good catch. Added the method in a second table after the first.
DONE